### DISCUSSION TITLE
400TB plotted, but no XCH rewards

### DISCUSSION THREAD

**Discussion by koolfur** [Chia-Network/chia-blockchain#5282] [Support]
Hi all,

I have properly connected all of my 40 harvesters to my farmer and have plotted 400TB of k-32. However, there is not even a single XCH drop for me after 1+ week. Could I been doing anything wrong?

2021-05-17T20:35:13.859 full_node chia.full_node.mempool_manager: INFO     It took 0.00400090217590332 to pre validate transaction

I am able to see everything I am farming in 'Last Attempted Proof'.

Thanks,
Chris

**Comment by SackofFluor** [comment]
did you check whether your plots are alright?
Did you see whether your look up times are fast enough ( <5sec?)

Did you check if your Mining address is correct?
With 400TB (if you had them running for 1 whole week) you were considered to need around 3 days. So 7 days is "long" but not outrageous.

My first coins where considered to need 16 days... i waited around 35...

**Reply by koolfur** [reply]
Yup I have done all the required plots checked.

The look up times are fast too, as shown on my initial post.

Here are my public keys:
2021-05-17T21:03:47.556  chia.plotting.check_plots        : INFO        Pool public key: 948c14a92024e8637a030eb790439112f1fefdc3a05857331e6791526ba62788f799ccf234352cae21f89eca01cb996b
2021-05-17T21:03:47.565  chia.plotting.check_plots        : INFO        Farmer public key: 8e9fbd6b38af23de7ac1ac2673710fdcfeef7cdf240db0aa7459f2b426a6d6d0605b0c51b0146f7cf73b7cfaad5d0dd3

Is there any place where I can check? Thanks

**Reply by lvcivs** [reply]
Also check your wallet address on chiaexplorer.com or chiastatus.com

**Reply by koolfur** [reply]
I tried entering my wallet address on both chiaexplorer and chiastatus.com but nothing is found

**Reply by firmicus** [reply]
.. same to me .... look at my post: https://github.com/Chia-Network/chia-blockchain/discussions/5292#discussion-3369394

**Reply by firmicus** [reply]
db corrupted ... I guess ....   waiting for reply from cimmunity ... let's see ..

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
